tests/test_options.py

```diff
@@ -106,9 +107,9 @@ def test_transparency():
 @pytest.mark.sphinx("html", testroot="image")
 def test_image(content: Sphinx, directives: List[Tag]):
     (img,) = directives
-    outdir = str(content.builder.outdir)
+    outdir = content.builder.outdir / "drawio" + os.sep
```

Comment:
If you're reapeating "outdir / drawio" in multiple places, could you try to create a `drawio_outdir` that you pass around instead?

Reply:
I am repeating this directory in two places - the module itself and the test.
The test should check if the directory is correct - testing against a variable from the implementation does not really help with that ;)